### DISCUSSION TITLE
msl error: user authentication data does not match entity identiy

### DISCUSSION THREAD

**Discussion by grega74** [CastagnaIT/plugin.video.netflix#1423] [General]
Hello

i get error msl error: user authentication data does not match entity identiy when I try to watch anything. Force L3 is activated. 

Any help would be nice :)

regards

**Comment by sheogorat96** [comment]
same only with new test version (login works, playback not) https://github.com/CastagnaIT/plugin.video.netflix/issues/1425#issuecomment-1264356850
With previous can't login :)

**Comment by grega74** [comment]
Same error with latest test build still ... Force L3 enabled (box is L1 certified but same situation when I leave L1)

log file: https://paste.kodi.tv/wadegagoye

With L1 enabled I get error: This title is not available to watch instantly

And even with previous version, it wasn't working for me - got also with L3: This title is not available to watch instantly - I did not change ESN, left it default for L1 and L3

Installed v4 of netflix for ATV and copied ESN from there, then I get error: Sorry your account cant be used on this device

**Reply by mrmuro** [reply]
For me this version of the plugin is still working: plugin.video.netflix-1.18.8+matrix

18.10 and 19 are not working. For 19 it my work, but only with the main account.
